packages/terra-toggle-section-header/examples/test-examples/DefaultToggleSectionHeader.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import ToggleSectionHeader from '../../lib/ToggleSectionHeader';
+
+export default () => (
+  <ToggleSectionHeader title="Default Toggle Section Header">
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
```

Comment:
Since this lorem ipsum text is used in several places, I'm wondering if we should store a constant somewhere. Do existing components that use lorem ipsum just copy/paste it everywhere?

Reply:
I don't think it's necessary, for me, visually seeing the `Lorem ipsum...` is enough to know that it's just supposed to be filler